DataFormats/TrackerCommon/interface/TrackerTopology.h

```diff
@@ -600,6 +600,13 @@ class TrackerTopology {
   bool hasField(const DetId &id, DetIdFields idx) const {
     return id.subdetId() == bits_per_field[idx].subdet;
   }
+
+  const PixelBarrelValues& pbVals() const { return pbVals_; }
+  const PixelEndcapValues& pfVals() const { return pfVals_; }
+  const TOBValues& tobVals() const { return tobVals_; }
+  const TIBValues& tibVals() const { return tibVals_; }
+  const TIDValues& tidVals() const { return tidVals_; }
+  const TECValues& tecVals() const { return tecVals_; }
```

Comment:
Ok, I was able to put TrackerTopology in the resulting root file after
* adding default constructor to TrackerTopology
* removing `const` from all TrackerTopology members
* adding dictionaries for TrackerTopology and all the inner structs (PixelBarrelValues etc)
* storing the TrackerTopology to another TTree (as the only branch and entry)
  * `TFileService::make<TrackerTopology>` complained that TrackerTopology does not inherit from TObject

From the usage point of view it is a bit unpleasant to have to convert the detid number to DetId object, e.g. `ttopo.pxbLayer(ROOT.DetId(303087632))`, but it's not that big of a deal.

Would persisting TrackerTopology be the preferred solution then?